modules/ROOT/pages/microsoft-azure-sb-connector.adoc

```diff
@@ -0,0 +1,190 @@
+|*Max. Messages to Receive*| Maximum number of messages to receive during the scheduled operation.
+|===
+
+== Use Case: Studio
+
+=== Get Queue list
+
+image::microsoft-azure-sb-flow.png[General]
+
+. Create a new Mule application in Studio, and select an HTTP Listener as a source in the new flow.
+. Add a new HTTP Listener Configuration global element:
+.. Enter the following values for the `Host` and `Port` fields:
++
+[%header%autowidth.spread]
+|===
+|Parameter |Value
+|*Host* |0.0.0.0
+|*Port* |8081
+|===
++
+.. Click *Save*.
+. Assign the Global configuration to the HTTP Listener.
+. In the *Path* field for the HTTP Listener path, enter `/list`.
+. Drag and drop a new *Microsoft Azure Service Bus* component on the flow.
+. Configure the Microsoft Azure Service Bus connector global element with its environment values.
+. Select the `List queues` operation in the dropdown option.
+. Save and run the project as a Mule Application.
+
+== Use Case: XML
+
+[source,code]
+----
+<?xml version="1.0" encoding="UTF-8"?>
+
+<mule xmlns:http="http://www.mulesoft.org/schema/mule/http" xmlns:microsoft-azure-service-bus="http://www.mulesoft.org/schema/mule/microsoft-azure-service-bus" xmlns:tracking="http://www.mulesoft.org/schema/mule/ee/tracking" xmlns="http://www.mulesoft.org/schema/mule/core" xmlns:doc="http://www.mulesoft.org/schema/mule/documentation"
+	xmlns:spring="http://www.springframework.org/schema/beans"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans-current.xsd
+http://www.mulesoft.org/schema/mule/core http://www.mulesoft.org/schema/mule/core/current/mule.xsd
+http://www.mulesoft.org/schema/mule/http http://www.mulesoft.org/schema/mule/http/current/mule-http.xsd
+http://www.mulesoft.org/schema/mule/microsoft-azure-service-bus http://www.mulesoft.org/schema/mule/microsoft-azure-service-bus/current/mule-microsoft-azure-service-bus.xsd
+http://www.mulesoft.org/schema/mule/ee/tracking http://www.mulesoft.org/schema/mule/ee/tracking/current/mule-tracking-ee.xsd">
+    <http:listener-config name="HTTP_Listener_Configuration" host="0.0.0.0" port="8081" doc:name="HTTP Listener Configuration"/>
+    <microsoft-azure-service-bus:connection-string-config name="Microsoft_Azure_Service_Bus__Connection_string" connectionString="${config.cs}" namespace="${config.namespace}" doc:name="Microsoft Azure Service Bus: Connection string"/>
+    <flow name="list-queue">
+        <http:listener config-ref="HTTP_Listener_Configuration" path="/" doc:name="HTTP"/>
+        <microsoft-azure-service-bus:list-queues config-ref="Microsoft_Azure_Service_Bus__Connection_string" doc:name="list queues"/>
+    </flow>
+</mule>
+
+----
+
+== Additional Information
+
+=== Queue/Subscription Listener Operations
```

Comment:
"/" is ambiguous -- please use "or" or "and"  to make meaning clearer.

Reply:
Done